docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -121,12 +121,22 @@ Going beyond that, you can specify an individual test method like this::
 Running the Selenium tests
 --------------------------
 
-Some tests require Selenium and a Web browser (Firefox, Google Chrome, or
+Some tests require Selenium(>=2.45) and a Web browser (Firefox, Google Chrome, or
 Internet Explorer). To allow those tests to be run rather than skipped, you must
 install the selenium_ package into your Python path and run the tests with the
-``--selenium`` option::
+``--selenium=<BROWSER>`` option.
+For example, using Firefox (``--selenium=firefox``)::
 
-   $ ./runtests.py --settings=test_sqlite --selenium admin_inlines
+    $ ./runtests.py --selenium=firefox admin_views.tests.SeleniumTests
```

Comment:
Let's not add "tests.SeleniumTests" -- the name or path might change (we need to reorganize the huge test file in admin_views/tests.py) and won't be necessary when https://code.djangoproject.com/ticket/25653 is completed.